architecture/sql-layer.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Architecture: SQL Layer
+summary: 
+toc: false
+---
+
+The SQL Layer of CockroachDB's architecture accepts client RPCs and sends them to the underlying transaction layer.
+
+<div id="toc"></div>
+
+## Overview
+
+To begin interacting with CockroachDB, developers need nothing more than a connection string that routes to a node and SQL statements. Every node in your cluster can receive SQL statements (either DDL or DML), which it then acts as "gateway node" for as it communicates with the rest of your cluster.
```

Comment:
We should simplify or explain DDL or DML as people don't immediately know what these are: https://stackoverflow.com/questions/2578194/what-is-ddl-and-dml

Reply:
Done.